src/microbe_stage/CompoundCloudPlane.cs

```diff
@@ -438,10 +516,11 @@ public void Update(float delta, Vector2 pos)
                 return;
 
             // Compound clouds move from area of high concentration to area of low.
-            Diffuse(0.007f, OldDensity, Density, delta);
+            Diffuse(delta);
+            ClearDensity();
```

Comment:
Is this clear density needed? It wasn't needed before, so this might be some extra unneeded processing, or did you change the logic so much that the clear here is needed?

Reply:
it was there before, just in the advect function

Reply:
Ah, okay. Perhaps this could be a place to get some performance by not clearing it and swapping between the different array references. I haven't looked at the full cloud code yet so I don't know if this is actually possible.

Reply:
you cant remove that, otherwise the compounds would multiply infinitely